src/main/java/org/milkteamc/autotreechop/AutoTreeChop.java

```diff
@@ -43,53 +43,29 @@ public class AutoTreeChop extends JavaPlugin implements Listener, CommandExecuto
 
     public static final Message ENABLED_MESSAGE = new MessageBuilder("enabled")
-            .withDefault("<positive>Auto tree chopping enabled.</positive>")
-            .withTranslation(Locale.GERMAN, "<positive>Auto-Baumf�llen aktiviert.</positive>")
-            .build();
+            .withDefault("<positive>Auto tree chopping enabled.</positive>").build();
     public static final Message DISABLED_MESSAGE = new MessageBuilder("disabled")
-            .withDefault("<negative>Auto tree chopping disabled.</negative>")
-            .withTranslation(Locale.GERMAN, "<negative>Auto-Baumf�llen deaktiviert.</negative>")
-            .build();
+            .withDefault("<negative>Auto tree chopping disabled.</negative>").build();
     public static final Message NO_PERMISSION_MESSAGE = new MessageBuilder("no-permission")
-            .withDefault("<negative>You don't have permission to use this command.</negative>")
-            .withTranslation(Locale.GERMAN, "<negative>Dazu hast du keine Berechtigung</negative>")
-            .build();
+            .withDefault("<negative>You don't have permission to use this command.</negative>").build();
     public static final Message HIT_MAX_USAGE_MESSAGE = new MessageBuilder("hitmaxusage")
-            .withDefault("<negative>You've reached the daily usage limit.</negative>")
-            .withTranslation(Locale.GERMAN, "<negative>Du hast dein Tageslimit zum Auto-Baumf�llen erreicht.</negative>")
-            .build();
+            .withDefault("<negative>You've reached the daily usage limit.</negative>").build();
     public static final Message HIT_MAX_BLOCK_MESSAGE = new MessageBuilder("hitmaxblock")
-            .withDefault("<negative>You have reached your daily block breaking limit.</negative>")
-            .withTranslation(Locale.GERMAN, "<negative>Du hast dein t�gliches Blocklimit zum Auto-Baumf�llen erreicht.</negative>")
-            .build();
+            .withDefault("<negative>You have reached your daily block breaking limit.</negative>").build();
     public static final Message USAGE_MESSAGE = new MessageBuilder("usage")
-            .withDefault("<positive>You have used the AutoTreeChop <current_uses>/<max_uses> times today.</positive>")
-            .withTranslation(Locale.GERMAN, "<positive>Du hast Auto-Baumf�llen heute <current_uses>/<max_uses> mal verwendet.</positive>")
-            .build();
+            .withDefault("<positive>You have used the AutoTreeChop <current_uses>/<max_uses> times today.</positive>").build();
     public static final Message BLOCKS_BROKEN_MESSAGE = new MessageBuilder("blocks-broken")
-            .withDefault("<positive>You have broken <current_blocks>/<max_blocks> blocks today.</positive>")
-            .withTranslation(Locale.GERMAN, "<positive>Du hast heute <current_blocks>/<max_blocks> Bl�cke gef�llt.</positive>")
-            .build();
+            .withDefault("<positive>You have broken <current_blocks>/<max_blocks> blocks today.</positive>").build();
```

Comment:
Its the MiniMessage standard to use opening tags for placeholders too. I totally agree that its ugly but since MiniMessage is already widely used (even PlotSquared) I had to stick with it. Its malformed XML and should definitely at least be a self closing tag but MiniMessage works in a way where a placeholder is at the same time an opening tag that does nothing with the content (everything that comes after it until a previously opened tag closes) except from inserting a value at the front.

I can understand if its reason enough not to use the Translations Framework but its nothing I can change